.pydevproject

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
let's not check in editing environment local files here. Particularly, because authors will not do python anyway.